src/components/tabs/Tabs.vue

```diff
@@ -12,27 +12,59 @@
 </template>
 
 <script>
-  import {createUniqueId} from '../../utils'
-
   export default {
```

Comment:
In general, I assume that this code fixes the problem when tabs are appearing/disappearing dynamically right? I that's the case, could you also add an example showing this capability to the docs? Do you mean something else as well when you said "reactivity problems"?

Reply:
Yes, make content of tab title dynamic and dynamically add/remove tabs.
I have added example in the doc.
Please check my last commit a6357b2